### ISSUE TITLE
Losses Wi-Fi connection when connecting Amazfit Bip to GB

### ISSUE THREAD

**Issue by lawrnz** [Freeyourgadget/Gadgetbridge#933]
When I connect my device (Amazfit Bip) to GadgetBridge my Wi-Fi connection losses, as if GB is interrupting the Wi-Fi connection. But when I turn off bluetooth, Wi-Fi connects again. 

#### Your wearable device is:
Amazfit Bip V0.1.0.27

#### Your android version is:
7.0

#### Your Gadgetbridge version is:
0.23.2

**Comment by lawrnz** [comment]
In addition, I didn't encounter this issue before when I was still on GB version 0.22.5

**Comment by lawrnz** [comment]
UPDATE: I've uninstalled GadgetBridge and tried connecting the device again with Mi Fit app via BT. Still getting the Wi-Fi issue. Wi-Fi is only affected whenever I turn on the BT on my device. This all started when I updated GB to 0.23.2.

**Comment by lawrnz** [comment]
After doing some research. I found out that the error was due to the recent android update that I got. I remembered receiving an update yesterday and proceeded to udpate my phone. I haven't turned on both Wi-Fi and BT yesterday after the update. It just so happened that I updated GB today and tested the BT connection while Wi-Fi is turned on (I don't usually turn on both at the same time), which I then assumed that the error was due to the GB update. Sorry about that guys. 

So there you go. Hopefully android or samsung can fix this on the next update thoug. Sigh.
